website/pages/api-docs/events.mdx

```diff
@@ -1,12 +1,12 @@
 ---
 layout: api
 page_title: Events - HTTP API
-sidebar_title: Events <sup>Beta</sup>
+sidebar_title: Events
 description: |-
   The /event endpoints are used to query for and stream Nomad events.
 ---
 
-# Events HTTP API <sup>Beta</sup>
+# Events HTTP API
```

Comment:
@drewbailey want to double check that we ought to remove the beta tag here

Reply:
LGTM